go.mod

```diff
@@ -9,58 +9,63 @@ replace github.com/grpc-ecosystem/grpc-gateway => github.com/TheThingsIndustries
 replace github.com/robertkrimen/otto => github.com/TheThingsIndustries/otto v0.0.0-20181129100957-6ddbbb60554a
 
 // github.com/blang/semver doesn't have a v3 semantic import.
-replace github.com/blang/semver v3.5.1+incompatible => github.com/blang/semver v0.0.0-20190414182527-1a9109f8c4a1
+replace github.com/blang/semver => github.com/blang/semver v0.0.0-20190414182527-1a9109f8c4a1
```

Comment:
Please only replace the exact offending version instead of all possible versions.

Reply:
All versions of `github.com/blang/semver` are offending. They're on `v3` by now, so we should be using semantic import `github.com/blang/semver/v3`, which they don't have. Since we don't accidentally want to revert to some old v1 version, I'm replacing all versions.